README.md

```diff
@@ -91,6 +91,9 @@ on the number of metrics that can be sent up at any given time.  `TelemetryMetri
 will send accumulated metric data at least every minute (configurable by the `:push_interval`
 option) or when the data cache has reached the maximum size that CloudWatch will accept.
 
+### Event Sampling
+You can control the rate at which your metrics are queued by configuring the `:sample_rate` option using a value between 0.0 and 1.0. A sample rate value of 0.0 will ensure no events are sent, whereas a sample rate value of 1.0 will ensure all events are sent. If you set the sample rate to 0.25, you would effectively queue 25% of your metrics.
```

Comment:
Might be worth mentioning that this would cause issues if you're using the "Sum", "Summary" (for sums), or "Counter" metrics just so users of the lib are aware (hopefully obvious, but I explicit > implicit).

Reply:
I was thinking about that as well, can you think of anyway to get around that? So you can get more of a complete picture, just less often?

Reply:
Naw, I think for something like Counters/Sums you'd just have to multiply by `1 / sample_rate` when looking at reports for a rough estimate of total values.  If users are just interested in Averages / Percentiles though and the volume of metric generation is high enough it should be fine.  The only issue is if the volume is low those numbers could be wildly off (if your sample rate is 10%, and there's 1 Counter event recorded for a specific period, the actual number of events could be 1, it could be 10, it could be 2,000,000).  If you just care about averages/percentiles over hours, and you know you'll have at least many thousands of events / minute, then 10% should (probably) be a fine sample rate.

Reply:
Added some documentation to clarify what you've mentioned here, along with your proposed resolutions for improving it.